diesel/src/lib.rs

```diff
@@ -131,11 +131,9 @@ extern crate bitflags;
 extern crate byteorder;
 #[macro_use]
 extern crate diesel_derives;
-#[doc(hidden)]
-pub use diesel_derives::*;
 
 #[macro_use]
-mod macros;
+pub mod macros;
```

Comment:
```
commit a4dfb76f09805e1a345bdd73ac12a88f8fc5a6b1
Author: Martell Malone <martellmalone@gmail.com>
Date:   Thu Aug 8 16:06:18 2019 -0700

    Move sql_function_proc to macros as hidden

    In order to do this we need to make macros public
    so that we can access macros version of sql_function_proc

    TODO: docs?
```